fiftyone/core/annotation/validate_label_schema.py

```
@@ -0,0 +1,529 @@
+        elif key == foac.TYPE:
+            invalid_date = (
+                isinstance(field, fof.DateField) and value != foac.DATE
+            )
+            invalid_datetime = (
+                isinstance(field, fof.DateTimeField) and value != foac.DATETIME
+            )
+            if invalid_date or invalid_datetime:
+                _raise_type_error(field, field_name, value)
+
+
+def _validate_dict_field_label_schema(collection, field_name, label_schema):
```

Pylint (4.0.3)

[convention] Function name "_validate_dict_field_label_schema" doesn't conform to '[a-z_]([a-z0-9_]{0,30})$' pattern

(C0103)